packages/arb-token-bridge-ui/src/pages/api/cctp/[type].ts

```diff
@@ -74,6 +74,46 @@ export type CompletedCCTPTransfer = PendingCCTPTransfer & {
   messageReceived: MessageReceived
 }
 
+function getMessageSents({
+  sender,
+  recipient,
+  searchString,
+  pageSize,
+  pageNumber
+}: Pick<
+  NextApiRequestWithCCTPParams['query'],
+  'searchString' | 'pageSize' | 'pageNumber'
+> & {
+  sender?: `0x${string}`
+  recipient?: `0x${string}`
+}) {
```

Comment:
I think (not 100% sure) on page load `sender` and `recipient` may be both undefined. Might be good to return an empty array in that case.

Reply:
if `walletAddress` is undefined, then this API is not called